textproc/epubcheck/Portfile

```diff
@@ -4,35 +4,38 @@ PortSystem              1.0
 PortGroup               java 1.0
 
 name                    epubcheck
-version                 1.2
+version                 4.2.2
 categories              textproc
 platforms               darwin
-license                 BSD 
+license                 BSD 3-Clause \"New\" or \"Revised\" License
 maintainers             nomaintainer
 supported_archs         noarch
 
-description             ePub Validator to validate ePub documents
+description             EPUB validator
 
-long_description        EpubCheck is a tool to validate IDPF  EPUB files. \
-                        It can detect many types of errors in EPUB. OCF \
-                        container structure, OPF and OPS mark-up, and \
-                        internal reference consistency are checked.
+long_description        EPUBCheck is a tool to validate the \
+                        conformance of EPUB publications against the \
+                        EPUB specifications. EPUBCheck can be run as a \
+                        standalone command-line tool or used as a Java \
+                        library.
 
-homepage                https://code.google.com/p/epubcheck/
-master_sites            googlecode
+homepage                https://github.com/w3c/epubcheck
+master_sites            https://github.com/w3c/epubcheck/releases/download/v${version}
```

Comment:
The GitHub portgroup should be used when obtaining distfiles from GitHub like this. I suggest looking at portfiles containing `github.setup` and `github.tarball_from releases` for examples.